controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -74,6 +74,7 @@
     public static final String TLS_TRUST_STORE;
     public static final String TOKEN_SIGNING_KEY;
     public static final int ACCESS_TOKEN_TTL_IN_SECONDS;
+    public static final String TLS_ENABLED_FOR_SEGMENT_STORE;
```

Comment:
https://github.com/pravega/pravega/blob/7f76baa4674f1e572cc0571bb22f505ae0583076/controller/src/main/java/io/pravega/controller/util/Config.java#L71
Here the config`TLS_ENABLED` of type boolean is for controller alone right ?

Reply:
Yes, that is right. Segment store has its own set of environment variables (in `ServiceConfig`).